client/lib/signup/step-actions/index.js

```diff
@@ -437,28 +439,110 @@ export function setOptionsOnSite( callback, { siteSlug, siteTitle, tagline } ) {
 	);
 }
 
-export function setStoreFeatures( callback, { siteSlug } ) {
+export async function setStoreFeatures( callback, { siteSlug }, stepProvidedItems, reduxStore ) {
 	if ( ! siteSlug ) {
 		defer( callback );
 		return;
 	}
 
-	//Hard-code the theme slug for now, we'll use design selection step in a future iteration.
+	try {
+		/*
+		 * Get the block pattern source for use in our new home page.
+		 * Ideally we'd pull this content from the patterns repo dynamically, but we don't
+		 * have a way to get raw block source.
+		 * Original pattern: https://dotcompatterns.wordpress.com/wp-admin/post.php?post=4348&action=edit
+		 */
+		const patternPost = `
+			<!-- wp:columns {"align":"wide"} -->
+			<div class="wp-block-columns alignwide">
+				<!-- wp:column -->
+				<div class="wp-block-column">
+					<!-- wp:image -->
+					<figure class="wp-block-image"><img alt="" /></figure>
+					<!-- /wp:image -->
+				</div>
+				<!-- /wp:column -->
+
+				<!-- wp:column {"style":{"spacing":{"padding":{"right":"10%","left":"5%"}}}} -->
+				<div class="wp-block-column" style="padding-right:10%;padding-left:5%">
+					<!-- wp:heading {"textAlign":"left","fontSize":"x-large"} -->
+					<h2 class="has-text-align-left has-x-large-font-size" id="item-name">${ translate(
+						'Item Name'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **39** times:
`translate( 'Item name' )` *ES Score: 9*